src/ubx.h

```diff
@@ -571,13 +569,17 @@ class GPSDriverUBX : public GPSHelper
 public:
 	GPSDriverUBX(Interface gpsInterface, GPSCallbackPtr callback, void *callback_user,
 		     struct vehicle_gps_position_s *gps_position,
-		     struct satellite_info_s *satellite_info);
+		     struct satellite_info_s *satellite_info,
+		     uint8_t dynamic_model);
```

Comment:
I thought because you have `RTCM mode force stationary dynamic model`, QGC gets it right anyway. I'm ok with either way.